app/models/content_item.rb

```diff
@@ -121,9 +121,14 @@ def gone?
 
   # Return a Hash of link types to lists of related items
   def linked_items
+    return links if already_expanded_links?
     LinkedItemsQuery.new(self).call
   end
 
+  def already_expanded_links?
+    links.to_s.match(/web_url/)
```

Comment:
maybe `links.has_key? "web_url"` instead? bit clearer what  the underlying data structure is.